tests/filecheck/lit.cfg

```diff
@@ -0,0 +1,12 @@
+import lit.formats
+import os
+
+config.test_source_root = os.path.dirname(__file__)
+snax_src = os.path.dirname(os.path.dirname(config.test_source_root))
+
+config.name = "SNAX"
+config.test_format = lit.formats.ShTest(preamble_commands=[f"cd {snax_src}"])
+config.suffixes = ['.test', '.mlir', '.py']
+
+config.substitutions.append(('XDSL_ROUNDTRIP', "snax-opt %s --print-op-generic --split-input-file | snax-opt --split-input-file | filecheck %s"))
+config.substitutions.append(("XDSL_GENERIC_ROUNDTRIP", "snax-opt %s --print-op-generic --split-input-file | filecheck %s --check-prefix=CHECK-GENERIC"))
```

Comment:
You are not using these in your tests, I think it's useful to have them here, but they should be tested/used